plugins/BEdita/Core/src/ORM/QueryFilterTrait.php

```diff
@@ -110,11 +110,11 @@ public function fieldsFilter(SelectQuery $query, array $options): SelectQuery
      *
      * @param \Cake\Database\Expression\QueryExpression $exp Current query expression
      * @param string $operator Filter operator
-     * @param string $field Filter field
+     * @param mixed $field Filter field
      * @param mixed $value Filter value
      * @return \Cake\Database\Expression\QueryExpression Operator query expression
      */
-    protected function operatorExpression(QueryExpression $exp, string $operator, string $field, mixed $value): QueryExpression
+    public function operatorExpression(QueryExpression $exp, string $operator, mixed $field, mixed $value): QueryExpression
```

Comment:
```suggestion
    public function operatorExpression(QueryExpression $exp, string $operator, ExpressionInterface|string $field, mixed $value): QueryExpression
```
adding `use` statement for `ExpressionInterface`.